external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentSequenceTopology.java

```diff
@@ -60,7 +61,7 @@ public static StormTopology buildTopology(String hdfsUrl){
                 .withSequenceFormat(new DefaultSequenceFormat("key", "sentence"))
                 .withRotationPolicy(rotationPolicy)
                 .withFsUrl(hdfsUrl)
-                .withConfigKey("hdfs.config")
```

Comment:
@tedxia 
if you change ```hdfs.config``` to ```topology.hdfs.config``` 
should the following several files  change too ?
[HdfsFileTopology.java](https://github.com/apache/storm/blob/master/external/storm-hdfs/src/test/java/org/apache/storm/hdfs/bolt/HdfsFileTopology.java#L82)
[simple_hdfs.yaml](https://github.com/apache/storm/blob/master/external/flux/flux-examples/src/main/resources/simple_hdfs.yaml#L74)
[hdfs_test.yaml](https://github.com/apache/storm/blob/master/external/flux/flux-core/src/test/resources/configs/hdfs_test.yaml#L82)